### ISSUE TITLE
Links between markdown pages don't work on DevOps pipeline build

### ISSUE THREAD

**Issue by PBoog** [MaxMelcher/AzureDevOps.WikiPDFExport#112]
**Expected behaviour**
On my laptop I clone my wiki, put the WikiPDFExport binary in the root and run an export for a specific sub folder of the wiki using the following instruction:
```
./azuredevops-export-wiki.exe --b --path .\Index\Application-Troubleshooting --attachments-path .\.attachments
```
This generates an export.pdf in which links between my markdown pages work.
For example in page ```/Index/Application-Troubleshooting/database.md``` I can specify a link like this
```
[Back to checklist](/Index/Application-Troubleshooting/checklist.md)
```
In the PDF it becomes a clickable link.

**Issue**
If I do the same thing in an Azure devops pipeline the export.pdf is generated but none of the links between different markdown pages work.

**Side note**
I have also noticed that running the WikiPDFExport binary without any parameters will correctly include images from .attachments on my laptop. However in the Azure pipeline I need to specify  ```--attachments-path ./.attachments``` to get the images to show up.

Looks like a difference in run time environment but no idea what it could be ;)
I have tried looking at the verbose output but that didn't help. Any thoughts on what could be causing this?

**Comment by PBoog** [comment]
Small update: I have tried a few different approaches in my devops pipeline
- Powershell build task on Windows 2022 build agent
- Bash build task on Ubuntu 22.04 build agent
- WIKI PDF Exporter build task on both windows and ubuntu build agents

For all of them the links between markdown pages are not working.

**Comment by PBoog** [comment]
Update:
To figure out what is going wrong I thought to add some log messages to see what happens. I'm using .NET 6.0 SDK with Visual studio 2022, publishing as a local Windows executable. I just pulled the code, opened the solution, added some log messages and compiled. Turns out my own compiled version does not have the issue.

So I suspect there is a difference in the way the release executable is getting built?
I'm stumped as to what is going on but happy to help reproduce or investigate.
In any case I have a workaround by using my own compiled version ;)

**Comment by MaxMelcher** [comment]
do you run it on windows or linux?

**Comment by PBoog** [comment]
My own build that does not have the issue is done on Windows. It works locally and in a Microsoft Azure DevOps pipeline.

For the release build that I downloaded from github:
- The windows build works locally on my laptop
- The windows build does not create links when running in a Microsoft Azure DevOps pipeline
- The linux build does not create links when running in a Microsoft Azure DevOps pipeline

**Comment by PBoog** [comment]
Thank you stale bot but this one is still open :)

**Comment by PBoog** [comment]
Thanks for keeping this on the backlog @MaxMelcher, if you need any further tests from my side please let me know.

**Comment by MaxMelcher** [comment]
Fixed in af0a226d787fe91314b2ab0a387ffbe754ba4d54
